ui/pages/swaps/view-quote/view-quote.js

```diff
@@ -674,143 +675,158 @@ export default function ViewQuote() {
     }
   }, [dispatch, viewQuotePageLoadedEvent, reviewSwapClickedTimestamp]);
 
-  return (
-    <div className="view-quote">
-      <div
-        className={classnames('view-quote__content', {
-          'view-quote__content_modal': disableSubmissionDueToPriceWarning,
-        })}
-      >
-        {selectQuotePopoverShown && (
-          <SelectQuotePopover
-            quoteDataRows={renderablePopoverData}
-            onClose={() => setSelectQuotePopoverShown(false)}
-            onSubmit={(aggId) => dispatch(swapsQuoteSelected(aggId))}
-            swapToSymbol={destinationTokenSymbol}
-            initialAggId={usedQuote.aggregator}
-            onQuoteDetailsIsOpened={quoteDetailsOpened}
-          />
-        )}
-
-        {showEditGasPopover && networkAndAccountSupports1559 && (
-          <EditGasPopover
-            transaction={{
-              userFeeLevel: swapsUserFeeLevel || 'high',
-              txParams: {
-                maxFeePerGas,
-                maxPriorityFeePerGas,
-                gas: maxGasLimit,
-              },
-            }}
-            minimumGasLimit={usedGasLimit}
-            defaultEstimateToUse="high"
-            mode={EDIT_GAS_MODES.SWAPS}
-            confirmButtonText={t('submit')}
-            onClose={onCloseEditGasPopover}
-          />
-        )}
+  const transaction = {
+    userFeeLevel: swapsUserFeeLevel || 'high',
+    txParams: {
+      maxFeePerGas,
+      maxPriorityFeePerGas,
+      gas: maxGasLimit,
+    },
+  };
 
+  return (
+    <>
```

Comment:
Is this fragment (`<>`) needed here? If we remove it, it will make the diff in this file a lot simpler

Reply:
Good point 👍